lib/image_sizes.php

```diff
@@ -0,0 +1,76 @@
+<?php declare(strict_types=1);
+
+class Image_Sizes {
+
+	public const SQUARE_SMALL = 'square-small';
+	public const SQUARE_MEDIUM = 'square-medium';
+	public const SQUARE_LARGE = 'square-large';
+	public const SIXTEEN_NINE = 'sixteen-nine';
+	public const SIXTEEN_NINE_SMALL = 'sixteen-nine-small';
+	public const SIXTEEN_NINE_LARGE = 'sixteen-nine-large';
+
+	/**
+	 * @var array<string, array<string, int|bool>>
+	 */
+	private array $sizes = [
```

Comment:
Just a note: Since it is a private item better to have public method get_sizes() that returns actual array.

Reply:
Also if you add `get_sizes` method it will be also good to have return result as filter e.g
`return apply_filters('ucsc/image/sizes', $this->sizes)`

Reply:
great idea @MlKilderkin . I'll get that added.